unit_test/batched/dense/Test_Batched_VectorArithmatic.hpp

```diff
@@ -6,10 +6,18 @@
 //       to ensure it is not included in these
 //       backends unit-test
 
-#if !defined(TEST_CUDA_BATCHED_DENSE_CPP) && \
-    !defined(TEST_HIP_BATCHED_DENSE_CPP) &&  \
-    !defined(TEST_SYCL_BATCHED_DENSE_CPP) && \
-    !defined(TEST_OPENMPTARGET_BATCHED_DENSE_CPP)
+#if !defined(TEST_CUDA_BATCHED_DENSE_SERIAL_OTHER_CPP) &&                   \
```

Comment:
It would be nice if this whole conditional could be made into a macro itself instead of copy pasting it a few times.